### PR TITLE
security

### PR DESCRIPTION
**Pull request by smriti548**
updated the security and solved assigned issue AC-417

### CODE REVIEW

File: Security [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Security
+OpenMRS is in contact with Security PS, an enterprise security consulting company.Security PS donated time to enhance the security of OpenMRS. Their application security experts found and privately reported software security weaknesses to the OpenMRS team. 
+The development team have analyzed Security PS’s results and have made and are making plans to address the discovered issues. 
+The org.openmrs.util.Security class provides basic encryption and decryption methods for use in the API.
+The security class used by Andro client is :
+java.lang.Object
+org.openmrs.util.Security and then public class that has to be implemented or added extends Object class.
```
Comment (shivtej1505) [inline Security]: No need to go into so much depth, remember, this is a guide is for any user who wants to use Android Client and he/she may not have knowledge about coding.

File: Security [hunk 2]
```diff
@@ -0,0 +1,93 @@
+Security
+OpenMRS is in contact with Security PS, an enterprise security consulting company.Security PS donated time to enhance the security of OpenMRS. Their application security experts found and privately reported software security weaknesses to the OpenMRS team.
```
Comment (shivtej1505) [inline Security]: Could you point me to the link where you found this?
Also, such information shouldn't be on a guide. Don't you think?

**Review by shivtej1505 — CHANGES_REQUESTED** [review]
Please squash your commit and change commit message. Check PR tips here: https://wiki.openmrs.org/display/docs/Pull+Request+Tips.

You didn't cover how the data exchanged with a server.

File: Security (hunk 2 above)
Comment (smriti548) [reply Security]: ok i'll change the content

File: Security (hunk 1 above)
Comment (smriti548) [reply Security]: But until and unless a user is not aware about the class used under which those encryption and decryption methods are being used for password security how could he use those features

Comment (shivtej1505) [reply Security]: You can just write that "we use _some algorithm_ which use _xxx bit key_ and fairly secure". You can also add 1-2 sentence about the working of the algorithm.

Comment (smriti548) [reply Security]: what about these methods? should i remove all of them?

File: Security [hunk 3]
```diff
@@ -0,0 +1,51 @@
+Security
```
Comment (f4ww4z) [inline Security:1]: Make this a header with `#`.

**Review by f4ww4z — CHANGES_REQUESTED** [review]
Please change the PR title to [AC-...] and follow the Pull Request tips as mentioned above.
https://wiki.openmrs.org/display/docs/Pull+Request+Tips

Can you also change the file to `security.md` (markdown file)?

File: Security (hunk 3 above)
Comment (smriti548) [reply Security:1]: sure will do it

**Comment by shivtej1505** [comment]
@smriti548 , are you still working on it?

**Pull request closed without merging**